deploy/charts/approver-policy/Chart.yaml

```diff
@@ -12,5 +12,5 @@ maintainers:
 sources:
 - https://github.com/cert-manager/approver-policy
 
-appVersion: v0.4.0
-version: v0.4.2
+appVersion: v0.0.0
+version: v0.0.0
```

Comment:
I don't want to bother updating these versions anymore, I also updated the release logic so there is just 1 version (same version for chart and application).